policy/modules/system/systemd.if

```diff
@@ -58,28 +69,603 @@ template(`systemd_role_template',`
 	# This domain is per-role because of the below transitions.
 	# See the systemd --user section of systemd.te for the
 	# remainder of the rules.
-	allow $1_systemd_t $3:process { setsched rlimitinh };
+	allow $1_systemd_t $3:process { setsched rlimitinh signal_perms };
+	allow $1_systemd_t self:netlink_kobject_uevent_socket create_socket_perms;
 	corecmd_shell_domtrans($1_systemd_t, $3)
 	corecmd_bin_domtrans($1_systemd_t, $3)
-	allow $1_systemd_t self:process signal;
+	allow $1_systemd_t self:process { getsched signal };
 
 	files_search_home($1_systemd_t)
 
+	init_link_key_template($1_systemd_t)
+
+	miscfiles_watch_localization($1_systemd_t)
+
+	mount_read_runtime_files($1_systemd_t)
+	mount_watch_runtime_files($1_systemd_t)
+	mount_watch_reads_runtime_files($1_systemd_t)
+
+	allow $3 $1_systemd_t:process signal;
 	# Allow using file descriptors for user environment generators
 	allow $3 $1_systemd_t:fd use;
 	allow $3 $1_systemd_t:fifo_file rw_inherited_fifo_file_perms;
+	allow $3 $1_systemd_t:unix_stream_socket { connectto rw_socket_perms };
```

Comment:
I wouldnt allow this here, but that is a bit subjective

Reply:
Still trying to reproduce the `connectto` access, but I added the latter to address these denials:
```
time->Fri Mar 12 16:50:44 2021
type=PROCTITLE msg=audit(1615585844.196:303): proctitle=2F62696E2F73797374656D63746C002D2D75736572007365742D656E7669726F6E6D656E74004
44255535F53455353494F4E5F4255535F414444524553533D756E69783A706174683D2F72756E2F757365722F313030302F627573
type=PATH msg=audit(1615585844.196:303): item=1 name="/lib64/ld-linux-x86-64.so.2" inode=655682 dev=fd:0a mode=0100755 ouid=0 ogid=0 
rdev=00:00 obj=system_u:object_r:ld_so_t:s0 nametype=NORMAL cap_fp=0 cap_fi=0 cap_fe=0 cap_fver=0 cap_frootid=0
type=PATH msg=audit(1615585844.196:303): item=0 name="/bin/systemctl" inode=655614 dev=fd:0a mode=0100755 ouid=0 ogid=0 rdev=00:00 ob
j=system_u:object_r:bin_t:s0 nametype=NORMAL cap_fp=0 cap_fi=0 cap_fe=0 cap_fver=0 cap_frootid=0
type=CWD msg=audit(1615585844.196:303): cwd="/home/shizune"
type=EXECVE msg=audit(1615585844.196:303): argc=4 a0="/bin/systemctl" a1="--user" a2="set-environment" a3="DBUS_SESSION_BUS_ADDRESS=u
nix:path=/run/user/1000/bus"
type=SYSCALL msg=audit(1615585844.196:303): arch=c000003e syscall=59 success=yes exit=0 a0=35dc1dd0 a1=35dc1e70 a2=35d3c130 a3=35dc21
00 items=2 ppid=1891 pid=1898 auid=1000 uid=1000 gid=1000 euid=1000 suid=1000 fsuid=1000 egid=1000 sgid=1000 fsgid=1000 tty=(none) se
s=2 comm="systemctl" exe="/bin/systemctl" subj=staff_u:staff_r:staff_t:s0-s0:c0.c1023 key=(null)
type=AVC msg=audit(1615585844.196:303): avc:  denied  { read write } for  pid=1898 comm="systemctl" path="socket:[8073]" dev="sockfs"
 ino=8073 scontext=staff_u:staff_r:staff_t:s0-s0:c0.c1023 tcontext=staff_u:staff_r:staff_systemd_t:s0-s0:c0.c1023 tclass=unix_stream_
socket permissive=1
----
time->Fri Mar 12 16:50:44 2021
type=AVC msg=audit(1615585844.202:304): avc:  denied  { getattr } for  pid=1898 comm="systemctl" path="socket:[8073]" dev="sockfs" in
o=8073 scontext=staff_u:staff_r:staff_t:s0-s0:c0.c1023 tcontext=staff_u:staff_r:staff_systemd_t:s0-s0:c0.c1023 tclass=unix_stream_soc
ket permissive=1
```